java/security/audit/spring-csrf-disabled.yaml

```diff
@@ -0,0 +1,8 @@
+rules:
+- id: spring-csrf-disabled
+  message: |
+    CSRF is disabled for this configuration. This is a security risk.
+    Do not do this unless there is a valid reason.
```

Comment:
I don't think "Do not do this unless there is a valid reason." is adding anything. Saying it's a security risk should let them know not to do it unless there's a valid reason.